docs/guides/index.markdown

```diff
@@ -3,22 +3,36 @@ layout: basic
 title: Guides
 ---
 
-### Guides
+## Guides
 
-#### User Guides
+### User Guides
 
 * [AeroGear Android User Guide](aerogear-android)
 * [AeroGear iOS 1.X User Guide](aerogear-ios)
 * [AeroGear iOS 2.X User Guide](aerogear-ios-2.X)
-* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
-* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-#### UnifiedPush Server
+### UnifiedPush Server
 
 * [AeroGear UnifiedPush Server resources](../unifiedpush)
 
+### Security
 
-#### Contributing to AeroGear
+* OTP
+  * [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+  * [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
+
+* OAuth2
+  1. [Basic steps](security/oauth2-basic-steps)
+  2. Client libraries
+
+     The following SDKs were crafted with the simplicity in mind and are constantly improving at each release.
```

Comment:
@lfryc to me is kinda of annoying have to access each page like iOS page, Android page...and etc. Into this way if people want to see all the SDKs available, they are grouped in a single place.

This is not a replacement to the downloads page, at the same time doesn't hurt to group them altogether.